src/html/htmlwin.cpp

```diff
@@ -1184,7 +1184,30 @@ void wxHtmlWindow::OnPaint(wxPaintEvent& WXUNUSED(event))
     }
 }
 
+void wxHtmlWindow::OnFocusEvent(wxFocusEvent& event)
+{
+    event.Skip();
 
+    // Redraw selection, because its background colour depends on
+    // whether the window has keyboard focus or not.
+
+    if ( !m_selection || m_selection->IsEmpty() )
+        return;
+
+    wxHtmlCell const* fromCell = m_selection->GetFromCell();
+    wxHtmlCell const* toCell = m_selection->GetToCell();
+    wxCHECK_RET(fromCell || toCell,
+                "Unexpected: selection is set but cells are not");
+    if ( !fromCell )
+        fromCell = toCell;
+    if ( !toCell )
+        toCell = fromCell;
+
+    wxPoint const tl = fromCell->GetAbsPos();
+    wxPoint const br = toCell->GetAbsPos() +
+                        wxPoint(toCell->GetWidth(), toCell->GetHeight());
```

Comment:
This changes absolutely nothing in the result, but it seems to me that using `wxSize` instead of `wxPoint` would be a bit more clear here.